apps/trading/components/accounts-container/sidebar-accounts-container.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { useT } from '../../lib/use-t';
+import { SwapContainer } from '../swap';
+import { WithdrawContainer } from '../withdraw-container';
+import {
+  AccountsContainer,
+  type AccountsContainerProps,
+} from '../accounts-container/accounts-container';
+import React from 'react';
+import { DepositContainer } from '@vegaprotocol/deposits';
```

Comment:
This should be the deposit-container from apps/trading/components. (you may need to rebase)